packages/terra-core-docs/src/terra-dev-site/doc/badge/About.1.doc.mdx

```diff
@@ -46,6 +47,7 @@ import Badge from 'terra-badge';
 <BadgeSize title="Size" />
 <BadgeIcon title="Icon" />
 <BadgeVisuallyHiddenText title="VisuallyHiddenText" />
+<BadgeInTable title="BadgeInTable" />
```

Comment:
Can you update the implementation notes on badge doc similar to this : https://github.com/cerner/terra-core/blob/d93c9e9ffe7ed8a3ef3f49f17497a7a017288db4/packages/terra-core-docs/src/terra-dev-site/doc/alert/About.1.doc.mdx#L57-L58

Reply:
Done https://github.com/cerner/terra-core/pull/3909/commits/fdb89cecb479d1e29495d6b4a10f08faa9e53b86